plugins/outputs/groundwork/README.md

```diff
@@ -36,3 +36,7 @@ This plugin writes to a [GroundWork Monitor][1] instance. Plugin only supports G
 * unitType - to use in monitoring contexts(subset of The Unified Code for Units of Measure standard). Supported types: "1", "%cpu", "KB", "GB", "MB".
 * warning  - to define warning threshold value.
 * critical - to define critical threshold value.
+
+## NOTE
+
+The current version of the plugin does not support metrics whose values are strings. Such metrics will be skipped and will not be added to the final payload.
```

Comment:
```suggestion
The current version of Groundworks does not support metrics whose values are strings. Such metrics will be skipped and will not be added to the final payload. You can find more context in this pull request: [#10255]( https://github.com/influxdata/telegraf/pull/10255)
```

Reply:
nitpick, but in case a future user stumbles on this then they will know to look at Groundworks for this feature.

Reply:
done